hail_scripts/reference_data/combine.py

```diff
@@ -81,34 +89,31 @@ def get_ht(dataset: str, reference_genome: str):
         },
     )
     ht = ht.transmute(**get_enum_select_fields(config.get('enum_select'), ht))
-    return ht.select(**{field_name: ht.row.drop(*ht.key)}).distinct()
+    ht = ht.select_globals(
+        **{
+            f'{dataset}_globals': hl.struct(
```

Comment:
I lifted the globals annotations up into `get_ht` because `version` needs to happen here and I wanted them all to be in the same place.  The `join` we do to combine the datasets also joins the globals, so we don't need to do a second step to combine the config.  Lastly, I think there is an existing bug where if we update two individual datasets together in code, and then run the code sequentially, the globals will reflect the in-code update before the second individual dataset is updated.

The downside of this approach is the naming of the variable.  We can't just name it `dataset` because we'll get a naming conflict with the row field.  And we also cannot nest inside a `datasets=hl.Struct(dataset=...)` struct as the join will mangle `datasets` into `datasets_1`, `datasets_2`, etc.